### ISSUE TITLE
Dissolver recipes do not use tags

### ISSUE THREAD

**Issue by mathgeniuszach** [al132mc/alchemistry#152]
Not sure if this is truly a bug, but the dissolver recipes from ingots to atoms do not use tags and instead use the respective ingot from chemlib only. This breaks compatibility with other mods ingots, and can be fixed very easily by replacing the respective code in `data\alchemistry\recipes\dissolver\ingot_*.json`:
```json
...
  "input": {
    "item": "chemlib:ingot_*"
...
```
with
```json
...
  "input": {
    "tag": "forge:ingots/*"
...
```

**Comment by ghost** [comment]
Good catch, yes this is something that needs to be fixed.

**Comment by ghost** [comment]
Fixed in 1.16.4-1, thank you
